frontend/src/v5/helpers/viewpoint.helpers.ts

```diff
@@ -20,11 +20,8 @@ import { generateViewpoint as generateViewpointV4, getNodesIdsFromSharedIds, toS
 import { formatMessage } from '@/v5/services/intl';
 import { dispatch, getState } from '@/v4/modules/store';
 import { isEmpty, isString } from 'lodash';
```

Comment:
Open an existing ticket and go to the groups tab. From there, create a smart group that should include a given number of objects. The group has no objects instead